src/TitleReceive/TitleReceiveList.js

```diff
@@ -52,11 +71,16 @@ const visibleColumns = [
   PIECE_COLUMNS.supplement,
 ];
 
+const ACCESSION_NUMBER_FIELD_NAME = (field, index) => `${field}[${index}].accessionNumber`;
+const BARCODE_FIELD_NAME = (field, index) => `${field}[${index}].barcode`;
+const CALL_NUMBER_FIELD_NAME = (field, index) => `${field}[${index}].callNumber`;
+
```

Comment:
To be honest, the usage of uppercase letters for function names a bit confuses me because they are usually used for constants or dictionaries. I'd prefer to name functions typically.

Reply:
Thanks @usavkov-epam !
I changed the function names to camelCase.